### DISCUSSION TITLE
How to creat Configuration file?

### DISCUSSION THREAD

**Discussion by chloejooo123** [akosbalasko/yarle#132] [Q&A] [answered]
I have no experience in coding but I need conversion between Evernote and RoamResearch. I don't know how can I create a configuration file and where to store it. Can someone give me some instructions?

**Comment by akosbalasko** [comment, marked as answer]
Hi @ChloeZhou1997 !
Just simple save this text to a file in any folder you want: 

```
{
    "enexSource": "/absolute-path-of-your-enex-dir/test-template.enex",
    "templateFile": "/absolute-path-of-your-template-dir/sampleTemplate.tmpl",
    "outputDir": "out",
    "keepOriginalHtml": false,
    "isMetadataNeeded": false,
    "isNotebookNameNeeded": false,
    "isZettelkastenNeeded": false,
    "plainTextNotesOnly": false,
    "skipLocation": false,
    "skipCreationTime": false,
    "skipUpdateTime": false,
    "skipSourceUrl": false,
    "skipWebClips": false,
    "skipTags": false,
    "useHashTags": true,
    "outputFormat": "StandardMD",
    "skipEnexFileNameFromOutputPath": false
    "monospaceIsCodeBlock": false,
    "keepMDCharactersOfENNotes": false,
}
``` 
Then you can edit it to set the preferred options and to set the absolute path of your enex file and your template. If there is no template set, then a default template will be applied. 
Finally, after installing Node described in the Readme, you can specify the path of the config file by --configFile switch. E.g. if your config file is convert.json in the current directory where you run the command, you would use --configFile convert.json or --configFile ./convert.json.

**Reply by iftekhar-codemaster** [reply]
WOW
